AI/Nullkiller/Analyzers/ObjectClusterizer.h

```diff
@@ -17,10 +17,10 @@ namespace NKAI
 
 struct ClusterObjectInfo
 {
-	float priority;
-	float movementCost;
-	uint64_t danger;
-	uint8_t turn;
+	float priority = 0;
+	float movementCost = 0;
+	uint64_t danger = 0;
+	uint8_t turn = 0;
```

Comment:
if zero itself doesn't matter, you could use `{}`

also `0.0f` for floats

Reply:
0 is more clear IMO than `{}`
But yeah, makes sense to use `0.0f` for floats

Reply:
well, the point with `{}` is "I want to have initialised memory but I don't really care which value that is, just some default one"